### DISCUSSION TITLE
I can't download any replays and overtake my place in top

### DISCUSSION THREAD

**Discussion by alexoshowo** [ppy/osu#20351] [Q&A]
### Type

Other

### Bug description

I basically can't download any kind of replays (except faling scores), and i have a lot of old RX scores, and when i beat a map with much bigger score - 
the game shows me my old RX score, and don't overtake it

### Version

osu! 2022.911.0

### Logs

[network.log](https://github.com/ppy/osu/files/9584197/network.log)
[performance.log](https://github.com/ppy/osu/files/9584199/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9584200/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9584201/updater.log)
[database.log](https://github.com/ppy/osu/files/9584202/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9584203/legacy-ipc.log)

**Comment by WindowsMeosu** [comment]
don't know about the 'not being able to download replays' thing but there are related issues with unable being able to override relax scores with NM:
related 1: https://github.com/ppy/osu/issues/20018
related 2: https://github.com/ppy/osu/issues/19993

**Reply by Theighlin** [reply]
Replays are not stored for now.
